commands/registerHandlers.go

```diff
@@ -0,0 +1,188 @@
+package commands
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Strum355/log"
+	"github.com/UCCNetsoc/discord-bot/api"
+	"github.com/UCCNetsoc/discord-bot/prometheus"
+	"github.com/bwmarrin/discordgo"
+	"github.com/dghubble/oauth1"
+	twitterApi "github.com/ericm/go-twitter/twitter"
+	"github.com/spf13/viper"
+)
+
+var (
+	// Twitter
+	twitterClient *twitterApi.Client
+)
+
+// Reaction on a user message
+type Reaction string
+
+const (
+	twitter Reaction = "🇹"
+)
+
+var (
+	commandsMap = make(map[string]func(context.Context, *discordgo.Session, *discordgo.InteractionCreate))
+	reactionMap = make(map[string]interface{}) // Maps message ids to content
+)
+
+type commandFunc func(context.Context, *discordgo.Session, *discordgo.InteractionCreate)
+
+func command(name string, function commandFunc) {
+	commandsMap[name] = function
+}
+
+// Register command handlers
+func RegisterHandlers(s *discordgo.Session) {
+	// TODO: Clean up repetition in registering commands and registering command handlers (e.g declaring command names both here and in the registerCommands.go)
```

Comment:
- [ ] TODO: Clean up repetition in registering commands and registering command handlers (e.g declaring command names both here and in the registerCommands.go).